applications/sandbox/autogen/autogen_prototype_example_comp.cpp

```diff
@@ -0,0 +1,28 @@
+#include "autogen_prototype_example_comp.hpp"
```

Comment:
Bit of a general question but shouldn't autopen files be gitignored? Seems a bit excessive and might cause conflicts more easily

Reply:
yeah ur right, they should be